doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_architecture_pm.rst

```diff
@@ -250,3 +244,105 @@ Power management benchmark
 **************************
 
 To benchmark the power consumption in *Idle* state, see :ref:`multicore_idle_test`.
+
+Power management optimization example
+*************************************
+
+This example focuses on a HID device use case (for example, a mouse)
```

Comment:
I would start that the mouse has two main application states:

1. Active - when the user is moving the mouse
2. Idle - after an inactivity timeout, until the user picks up the device again

The workflow below applies to the "Active" application state.